jsonschema-generator/src/main/java/com/github/victools/jsonschema/generator/FieldScope.java

```diff
@@ -112,6 +113,14 @@ public MethodScope findGetter() {
      * @return public getter from within the field's declaring class
      */
     private MethodScope doFindGetter() {
+        if (this.getMember().getType().isInstanceOf(Supplier.class)) {
```

Comment:
**suggestion**: While this achieves the specific use-case you're after, it constitutes a breaking change to this library. I suggest making this dependent on the `Option.FLATTENED_SUPPLIERS` at the least.

Plus: the supplier's functional interface implementation is not technically a field's getter. The actual getter could have annotations that may be relevant to consider too, which would get lost here now, wouldn't they?

I'm not sure this is the right place for this.

Reply:
Since this is just about picking up the annotation on the `get()` method, I'd rather see this as the responsibility of the `Option.FLATTENED_SUPPLIERS`' implementation. It should be part of the reusable type definition rather than having to be inlined for each property.

Reply:
If possible, this should be handled within the `FlattenedWrapperModule`. May need to indicate somehow which method is deemed relevant for this.